Steg 1

Dyrker opened the pull request.

Dyrker committed:
Ny design och tasks tillagda.

UI ser nu typ ut som figma och en ny class som heter TaskListItem är tillagd.

Dyrker committed:
Städat och laggt till ny view

Har även tagit bort den där konstiga state klassen, kanske kommer få ångra det sen hehe

TIGStudent commented:
Din kod ser bra ut och Jag har inga problem att läsa kod. Du har inga kommentarer för att förklara koden, men jag ser heller inget behov av kommentarer då koden är läsbar utan dem.
 
Det enda som jag tycker saknas är den lilla menyn för att filtrera listan efter "Alla", "Klara" och "Ej klara".
 
Jag kan se att du har satt färgen på appbar direkt i coden för appbaren, det kan vara bättre att sätta den färgen i theme i stället.
Ser att du har separerat ur koden för classen TaskListItem. Det är inget krav men jag ser fler delar av main.dart som kan separeras ut i sina egna filer för att organisera koden.

Appen följer wireframen ganska bra, men du behöver inte vara helt strikt med den. Du kan lägga till din egen personliga touch i designen om du vill, men det är säkert något du redan vet. Det viktigaste är att se till att de tre viktiga delarna finns med: startsidan, en separat sida för att lägga till saker (inte en popup), och filtreringslistan.

Bra jobbat och lycka till med att erövra Paraguay!

Heijdenberg reviewed (approved):
Din kod ser bra ut och Jag har inga problem att läsa kod. Du har inga kommentarer för att förklara koden, men jag ser heller inget behov av kommentarer då koden är läsbar utan dem.
 
Det enda som jag tycker saknas är den lilla menyn för att filtrera listan efter "Alla", "Klara" och "Ej klara".
 
Jag kan se att du har satt färgen på appbar direkt i coden för appbaren, det kan vara bättre att sätta den färgen i theme i stället.
Ser att du har separerat ur koden för classen TaskListItem. Det är inget krav men jag ser fler delar av main.dart som kan separeras ut i sina egna filer för att organisera koden.

Appen följer wireframen ganska bra, men du behöver inte vara helt strikt med den. Du kan lägga till din egen personliga touch i designen om du vill, men det är säkert något du redan vet. Det viktigaste är att se till att de tre viktiga delarna finns med: startsidan, en separat sida för att lägga till saker (inte en popup), och filtreringslistan.

Bra jobbat och lycka till med att erövra Paraguay!

Dyrker committed:
brytit ut pages ur main, skapat filter dialog, satt theme

HomePage och AddTask page ligger nu i egna filer och en dialog för filtrering av tasks är skapad, men innehåller ingen funktionalitet. Appbar och button färger sätts nu av theme i material build.
